src/app/profile/profile.component.html

```diff
@@ -15,21 +15,14 @@ <h2>Profile</h2>
         <input type="text" id="email" formControlName="email">
         <span class="error-message" *ngIf="controlHasError('email', 'invalidEmail')">Must include "@" symbol</span>
     </p>
+
     <hr>
-    <h2>Phone Numbers</h2>
-    <div formArrayName="phoneNumbers" *ngFor="let _ of PhoneNumbersArray.controls; let phoneGroupIndex = index">
-        <div class="phone-number" [formGroupName]="phoneGroupIndex">
-            <p>
-                <label for="number">Number {{phoneGroupIndex}}</label>
-                <input type="text" id="number{{phoneGroupIndex}}" formControlName="number">
-            </p>
-            <button class="remove-button" (click)="removeNumber(phoneGroupIndex)">Remove</button>
-        </div>
-    </div>
-    <button type="button" (click)="addFormControl()">add</button>
+
+   <phone-numbers-list [parentForm]="profileForm"></phone-numbers-list>
```

Comment:
Beautiful thing when you see a bunch of markup shring down into a single line. :)

A couple things here.  

- Instead of exposing the entire parent form into the child, only pass the form array the child needs.   It will communicate good code design and intent.  If you pass in the whole form, another engineer could be lured into doing something yucky with the whole form from the child component.

- For variable naming, I'd name "parentForm" to "profileGroup".  Gives the variable name a little more meaning.